code/modules/surgery/organs/lungs.dm

```diff
@@ -326,4 +334,4 @@
 	cold_level_1_damage = -COLD_GAS_DAMAGE_LEVEL_1 //They heal when the air is cold
 	cold_level_2_damage = -COLD_GAS_DAMAGE_LEVEL_2
 	cold_level_3_damage = -COLD_GAS_DAMAGE_LEVEL_3
```

Comment:
these values need changing; as I pointed out earlier; just doing this refactor buffs Drask's ability to heal from cold fairly dramatically. They should be:

1 = -0.75
2 = -2.25
3 = -4.5

Reply:
Testing now. Technically they healed equal amounts for each before, but the burn damage was set at half to account for their species BURN_mod multiplying the amount